src/main/haskell/kore/src/Kore/Step/Function/EvaluationStrategy.hs

```diff
@@ -1,7 +1,7 @@
 {-|
 Module      : Kore.Step.Function.EvaluationStrategy
 Description : Various strategies for evaluating functions.
-Copyright   : (c) Runtime Verification, 2018
+Copyright   : (c) Runtime Verification, 2019
```

Comment:
I think here you're supposed to put `2018-2019`

Reply:
This was just fixing a mistake, I created this file a few days ago.